Assets/Mapzen/VectorData/Formats/MvtFeature.cs

```diff
@@ -186,62 +181,90 @@ public override bool HandleGeometry(IGeometryHandler handler)
             switch (feature.Type)
             {
                 case PbfGeomType.Point:
-                    if (decoder.AdvanceCommand() && decoder.Command == CommandType.MoveTo)
+                    while (decoder.HasData())
                     {
-                        for (int i = 0; i < decoder.Repeat; i++)
+                        decoder.AdvanceCommand();
+                        Debug.Assert(decoder.Command == CommandType.MoveTo && decoder.Repeat > 0);
```

Comment:
Any reason not to use `Assert.IsTrue` from `using UnityEngine.Assertions`?

Reply:
I went back and forth a lot with how to handle these. I've been treating the code outside of the `Mapzen.Unity` namespace as not Unity-specific, so I was reluctant to use the Unity assertions library here. I first tried `Debug.Assert` from `System.Diagnostics`, but found that these do nothing in the Unity runtime. However, there is a `Debug.Assert` with the same signature in `UnityEngine`, so I was able to switch the assertion implementation with the `#if` at the top such that it uses `UnityEngine` assertions when compiling for Unity and otherwise uses `System.Diagnostics` assertions.